charts/datadog-crds/values.yaml

```diff
@@ -9,6 +9,12 @@ crds:
   datadogAgents: false
   # crds.datadogMonitors -- Set to true to deploy the DatadogMonitors CRD
   datadogMonitors: false
+  # crds.extendedDaemonSetReplicaSets -- Set to true to deploy the ExtendedDaemonSetReplicaSets CRD
```

Comment:
For this I followed the same pattern used for the operator CRDs. However, in this case I wonder if we could simplify this to just one option that installs all the EDS CRDs or none.

Reply:
yes I think we can have only one settings for the 3 EDS crds, because it is not possible to disable one of them in the EDS-controller.

Reply:
Fixed :+1: